cmd/run.go

```diff
@@ -50,8 +50,18 @@ func newRunCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// runCmd represents the run command
 	var runCmd = &cobra.Command{
 		Use:   "run",
-		Short: "Run the local Appsody environment for your project",
-		Long:  `This starts a docker based continuous build environment for your project.`,
+		Short: "Run your Appsody project in a containerized development environment.",
+		Long: `Run the local Appsody environment, starting a container based continuous build environment for your project.
+		
+Run this command from the root directory of your Appsody project`,
+		Example: `  appsody run
+  Runs your project in a containerized development environment.
+
+  appsody run --interactive
+  Runs your project in a containerized development environment, and attaches the standard input stream to the container. This allows you to interact with the processes within the container.
```

Comment:
"This" at the beginning of a sentence can be unclear. Is it accurate to say "You can use the standard input stream to interact with processes inside the container. "?